### PR TITLE
build: Merge wasmd 0.19.0

### PR DESCRIPTION
**Pull request by loloicci**
## Description

Merging CosmWasm/wasmd/x/wasm 's version 0.19.0

## Motivation and context

We will upgrade cosmwasm and wasmvm to version 0.16.x. And, this upgrade needs to use them.

## How has this been tested?

Unit tests in x/wasm.

## Checklist:

- [ ] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -30,7 +30,7 @@ require (
 	github.com/line/iavl/v2 v2.0.0-init.1.0.20220215225951-cb11c91d8857
 	github.com/line/ostracon v1.0.3
 	github.com/line/tm-db/v2 v2.0.0-init.1.0.20220121012851-61d2bc1d9486
-	github.com/line/wasmvm v0.14.0-0.8.0
+	github.com/line/wasmvm v0.16.3-testing
```
[the remark this answers is not in the document]
Comment (loloicci) [reply go.mod:33]: OK, after tagging to wasmvm (needs tagging to cosmwasm after merging https://github.com/line/cosmwasm/pull/161), I will update it.

File: x/wasm/keeper/handler_plugin_test.go [hunk 2]
```diff
@@ -115,7 +117,7 @@ func TestSDKMessageHandlerDispatch(t *testing.T) {
 	}{
 		"all good": {
 			srcRoute: sdk.NewRoute(types.RouterKey, capturingRouteFn),
-			srcEncoder: func(sender sdk.AccAddress, msg json.RawMessage, customEncodeRouter types.Router) ([]sdk.Msg, error) {
```
[the remark this answers is not in the document]
Comment (loloicci) [reply x/wasm/keeper/handler_plugin_test.go:118]: You are correct. But this patch contains various changes around these functions, so I chose to delete it temporarily and re-enable it after porting the Token/Collection module.

File: docs/proto/proto-docs.md [hunk 3]
```diff
@@ -0,0 +1,1116 @@
+<!-- This file is auto-generated. Please do not modify it yourself. -->
```
[the remark this answers is not in the document]
Comment (loloicci) [reply docs/proto/proto-docs.md:1]: nice found

**Comment by brew0722** [comment]
we need remove the `--source `and `--builder` option using the `lbm tx wasm store` and `lbm tx wasm sotre-instantiate`.

> Error: accepts 1 arg(s), received 0
Usage:
  lbm tx wasm store [wasm file] --source [source] --builder [builder] [flags] 

> Usage:
>   lbm tx wasm store-instantiate [wasm file] [json_encoded_init_args] --source [source] --builder [builder] --label [text] --admin [address,optional] --amount [coins,optional] [

**Comment by loloicci** [comment]
@brew0722 
> store-instantiate still has --source and --builder options. we need remove this.

Thanks. It is in only help message and does not work now. I removed them from help messages.

**Comment by brew0722** [comment]
In general, LGTM. please add to the Unreleased section in the changelog.

**Review by zemyblue — APPROVED** [review]

**Pull request merged**